core/parameters/src/config_store.rs

```diff
@@ -135,9 +155,13 @@ impl RuntimeConfigStore {
         match chain_id {
             near_primitives_core::chains::TESTNET => {
                 let genesis_runtime_config = RuntimeConfig::initial_testnet_config();
-                Self::new(Some(&genesis_runtime_config))
+                Self::new(Some(&genesis_runtime_config), false)
             }
-            _ => Self::new(None),
+            near_primitives_core::chains::BENCHMARKNET => {
+                let genesis_runtime_config = RuntimeConfig::initial_benchmarknet_config();
```

Comment:
Why do we still need to overwrite the base config? In my understanding, the diff that we apply on top of last protocol version should be enough to overwrite the parameters that we care about.